app/views/foreman_tasks/tasks/show.html.erb

```diff
@@ -1,94 +1,117 @@
 <% stylesheet 'foreman_tasks/tasks' %>
 
-<script>
-  if (typeof taskProgressReloader === 'undefined') {
-    var taskProgressReloader = {
-      timeoutId: null,
-      formId: null,
-      reload: function () {
-        // we need different reload mechanism for two-pane and non-two-pane
-        if (typeof currentTwoPaneTask !== 'undefined') {
-          if($('.two-pane-right').length && $('form#' + taskProgressReloader.formId).length) {
-            taskProgressReloader.timeoutId = null;
-            two_pane_open(currentTwoPaneTask);
+<%= breadcrumbs(
+  items: [
+    {
+      caption: _('Tasks'),
+      url: url_for(foreman_tasks_tasks_path)
+    },
+    {
+      caption: truncate(format_task_input(@task), :length => 50)
+    }
+  ],
+  name_field: 'action',
+  resource_url: foreman_tasks_api_tasks_path,
+  switcher_item_url: foreman_tasks_task_path(:id => ':id')
+) %>
+
+
+<div class="task-details">
+  <script>
```

Comment:
It's showing such a big diff but I just moved the script into `.task-details` and updated `taskProgressReloader.reload`.

Reply:
Sorry i forgot to mention it.
I replaced the call to `two_pane_open` with an `ajax` call,
so instead doing full reload, i'm getting from the server only the `div.class-details` and replace the current one by the new one.
We need to have the script inside the `div.class-details` so it will get reloaded when the div replaced.
It mostly needed to reapply the event handlers.